tools/notification-configuration/README.md

```diff
@@ -0,0 +1,7 @@
+# About
+
+This directory contains sources of tools implementing the
+"Subscribe to AzureDevOps notifications through CODEOWNERS file" feature.
+
+To learn more about it, including links to relevant ADO pipelines, please see
+the [feature wiki documentation](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/63/Subscribe-to-Notifications).
```

Comment:
As mentioned in the general comments I think we can probably move most of not all of those docs out to this MD file.

Reply:
@weshaggard I was thinking that to maintain ergonomics of editing, and article structure/form/layout/style, it perhaps would be better to move it not to an .md file, but to the [`azure-sdk-tools` repo wiki, which is currently empty](https://github.com/Azure/azure-sdk-tools/wiki). I want to give it a shot - migrate this article to the repo wiki, and then over time migrate more articles there, where it makes sense. 

Thoughts?

Reply:
I had a chat with @weshaggard . We avoid GitHub wikis because we want for the tooling docs to live as close to the tooling itself as possible. This aids discoverability, both by making the docs being right next to sources, as well as minimizing the risk that when the source moves, the docs are not moved as well. It also helps with preventing stale docs (because developers will see the docs and hopefully remember to update them) or obsolete docs (because when the source is deleted, hopefully the developer will see the docs and delete them too).

Hence, as originally recommended, I will convert the internal wiki doc to an `.md` file.